go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -618,6 +619,45 @@ func BenchmarkPlanner(b *testing.B) {
 	}
 }
 
+func BenchmarkSelectVsDML(b *testing.B) {
+	vschema := &vschemaWrapper{
+		v:             loadSchema(b, "schema_test.json"),
+		sysVarEnabled: true,
+		version:       V3,
+	}
+
+	var dmlCases []testCase
+	var selectCases []testCase
+
+	for tc := range iterateExecFile("dml_cases.txt") {
+		if tc.output2ndPlanner != "" {
```

Comment:
I don't think you should check `output2ndPlanner`. you should just use `output`

Reply:
Fixed!